src/Package/MSBuild.VSSetup/files.swr

```diff
@@ -5,6 +5,10 @@ package name=Microsoft.Build
         vs.package.chip=neutral
         vs.package.language=neutral
 
+vs.dependencies
+  vs.dependency id=Microsoft.VisualStudio.NuGet.BuildTools
```

Comment:
This VSIX will (at least should) only be installed in Build Tools SKU. 

Is that the only scenario in question?